binding.gyp

```diff
@@ -2,7 +2,21 @@
     "targets": [
         {
             "target_name": "roboticscape",
-            "sources": [ "rc-node-bindings.cc" ],
+	    "variables": {
+		   'RC_version': '<!(rc_version)'
+            },
+            "conditions": [
```

Comment:
Please use 0.4.4 and obsolete use for the older library. In fact, James has recently made a 1.0.0 release: https://github.com/StrawsonDesign/librobotcontrol/releases/tag/V1.0.0

It should be 0.4.4 compatible, I hope.